.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "cSpell.words": ["adal", "teamspace", "uninitialize"],
+  "cSpell.words": ["adal", "ipados", "teamspace", "uninitialize"],
```

Comment:
>ipados [](http://example.com/codeflow?start=28&length=6)

This is gross but it is the casing that is used in our enum so I guess we have to accept it.

Reply:
what do you mean I love ipa dos